service_client.go

```diff
@@ -114,6 +114,8 @@ func (client *ServiceClient) setMicroversionHeader(opts *RequestOpts) {
 		opts.MoreHeaders["X-OpenStack-Nova-API-Version"] = client.Microversion
 	case "sharev2":
 		opts.MoreHeaders["X-OpenStack-Manila-API-Version"] = client.Microversion
+	case "volume":
+		opts.MoreHeaders["X-OpenStack-Volume-API-Version"] = client.Microversion
```

Comment:
No problem.

My understanding is that it will act like the other services (Nova, Manila) that implement microversions. A user won't be able to use a feature that requires a specific microversion. So the request won't be outright rejected, but certain features won't be usable.